### DISCUSSION TITLE
Packard Bell PB570 "Hillary" motherboard

### DISCUSSION THREAD

**Discussion by TheFlyingScotsman** [86Box/86Box#1782] [Ideas]
Please could you add support for the Packard Bell PB570 (a.k.a. Hillary) motherboard?  I have the ROM if needed.  This board has integrated Cirrus Logic CL-GD5434.  Thanks.

**Comment by jriwanek** [comment]
Providing the ROM would make it more likely, along with datasheets, dumps of the kbc, pictures of the board so we can identify the chipset too.

**Comment by headsetguy37** [comment]
The chipset was the Triton 82437FX, Super I/O was the SMsC FDC37C665GT. Don't know about the keyboard controller, but here is a link to the ROM set (used two BIOS ROMs): https://mega.nz/file/TwInBaKS#MPfl37QCLZ-WvDvzmnPem0pSSlb8D-NmCboi-KBdaio

**Reply by OBattler** [reply]
According to the UKT Support site, the chipset was actually 430NX (Neptune) + SIO.ZB, but yes, with SM(S)C FDC37C665GT Super I/O chip. It also had a RZ-1000 IDE controller. The keyboard controller is most likely Phoenix MultiKey/42 on an Intel 8742, but I'm just going by other boards in the same era.

**Reply by TheVirtualComputer** [reply]
Just a correction: The PB570 did indeed use the Intel Triton chipset, but for some reason the website mentioned above seems to have misidentified the website (it appears to have confused it with the chipset from the PB540 board which actually did use the Neptune chipset), but this link here correctly identifies the PB570 board's chipset as being an Intel Triton chipset: https://www.oocities.org/sjg/hillary.html

In addition, you will notice that the PB570 BIOS won't run on anything less than an Intel Triton chipset (I've even tried this myself) which means that it requires at least a Triton chipset even for it to work at all.

**Reply by OBattler** [reply]
This is now being investigated because it's quite an interesting situation. So far, I've done I/O logging on your PB570 BIOS, and yes, it's 430FX+PIIX but with the National Semiconductors PC87306 Super I/O chip. And ye, the BIOS does say Hillary. However, my suspicion is now that someone somewhere misidentified the PB570 as being Hillary, because an archived version of Packard Bell's own website says 430NX+SIO.ZB with the SM(S)C FDC73C665 Super I/O chip, so it must be a different Intel board.

Link to the archived Packard Bell site: https://web.archive.org/web/20000303140106/http://support.packardbell.com:80/hardware/mb/pentium/pb570/PB570OVW.asp .
